### PR TITLE
ISPN-10811 Provide utility to convert caches configuration between JS…

### PR DESCRIPTION
**Pull request by gustavonalle**
https://issues.jboss.org/browse/ISPN-10811

### CODE REVIEW

File: documentation/src/main/asciidoc/topics/rest_api_v2.adoc [hunk 1]
```diff
@@ -95,6 +95,18 @@ include::rest_examples/get_v2_configuration.adoc[]
...
 |===
 
+[[rest_v2_cache_config_convert]]
+=== Converting Cache Configurations
+
+To convert a certain existing cache configuration that is in XML format to JSON, invoke:
+
+[source,options="nowrap",subs=attributes+]
+----
+POST /rest/v2/caches?action=toJSON
```
Comment (tristantarrant) [inline documentation/src/main/asciidoc/topics/rest_api_v2.adoc]: I don't understand the need for a special action. Couldn't this behaviour be triggered by the Accept-Content  header ?

Comment (gustavonalle) [reply documentation/src/main/asciidoc/topics/rest_api_v2.adoc]: This  method is not touching any cache or existing config, it is converting something POSTed as XML to JSON

Comment (tristantarrant) [reply documentation/src/main/asciidoc/topics/rest_api_v2.adoc]: Sorry I completely misunderstood that :)

**Comment by gustavonalle** [comment]
I will clarify in the docs in the next PR

**Pull request merged**